includes/Runtime.php

```diff
@@ -55,7 +57,8 @@ public function prepareRuntime() {
 			'restUrl' 	=> \esc_url_raw( \get_home_url() . '/index.php?rest_route=' ),
 			'restNonce'			=> wp_create_nonce('wp-rest'),
 			'isWoocommerceActive' 	=> is_plugin_active('woocommerce/woocommerce.php'),
-			...$sdk
+			...$sdk,
+			...$sdkProps
```

Comment:
@ramyakrishnai The PHP spread operator syntax was implemented in PHP 7.4. Since our brand plugin code is expected to work as far back as PHP 7.1, we'll need to remove those references (see https://github.com/bluehost/bluehost-wordpress-plugin/blob/main/.github/workflows/cypress-matrix.yml#L25-L31).